practice-app/src/main/templates/create_cocktail.html

```diff
@@ -0,0 +1,42 @@
+<html>
```

Comment:
I hope you'll convert them into .vue so that it will be easier for creating frontend as a group